crud/common/utils.lua

```diff
@@ -375,6 +386,52 @@ function utils.get_bucket_id_fieldno(space, shard_index_name)
     return bucket_id_index.parts[1].fieldno
 end
 
+-- NOTE: it is like get_bucket_id_fieldno()
+-- but it uses space's format instead of index
+function utils.get_fieldno_by_name(space_format, field_name)
+    field_name = field_name or 'bucket_id'
```

Comment:
Default value is chosen quite strange. It looks better to omit default at all and pass field_name always explicitly.

Reply:
removed default value